2018-10-23 15:35:02,554 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,556 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,556 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,558 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,558 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,558 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,561 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,561 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,561 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,561 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,563 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,563 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,563 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,563 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,563 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,565 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,565 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,565 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,565 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,565 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:02,565 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:35:07,546 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 15:35:07,546 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 15:35:07,546 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 15:35:07,546 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 15:35:07,546 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 15:35:07,546 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 15:35:07,546 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 15:35:07,546 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 15:35:07,546 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 15:35:07,546 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 15:35:07,546 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 15:35:07,546 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 15:36:50,299 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,301 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,301 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,302 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,302 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,302 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,305 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,305 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,305 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,305 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,307 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,307 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,307 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,307 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,307 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,309 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,309 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,309 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,309 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,309 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:50,309 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:36:55,241 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 15:36:55,241 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 15:36:55,241 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 15:36:55,241 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 15:36:55,241 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 15:36:55,241 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 15:36:55,241 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 15:36:55,241 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 15:36:55,241 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 15:36:55,241 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 15:36:55,241 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 15:36:55,241 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 15:37:18,854 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,856 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,856 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,858 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,858 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,858 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,861 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,861 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,861 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,861 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,863 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,863 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,863 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,863 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,863 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,865 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,865 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,865 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,865 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,865 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:18,865 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 15:37:22,484 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 15:37:22,484 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 15:37:22,484 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 15:37:22,484 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 15:37:22,484 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 15:37:22,484 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 15:37:22,484 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 15:37:22,484 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 15:37:22,484 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 15:37:22,484 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 15:37:22,484 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 15:37:22,484 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 15:39:31,025 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 15:39:31,025 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 15:39:31,025 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 15:39:31,025 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 15:39:31,025 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 15:39:31,025 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 15:39:31,026 - ERROR: Bioagents - ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 440, in send
    timeout=timeout
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 639, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/util/retry.py", line 357, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 82, in respond_find_drugs_for_mutation_dataset
    result = self.BA.find_drugs_for_mutation_dataset(gene_list, dataset_arg.lower())
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_agent.py", line 221, in find_drugs_for_mutation_dataset
    for row in O.query().V(list(mut_samples[g])).in_("responseFor").mark("a").out("responseTo").mark("b").select(["a", "b"]):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/gripql/query.py", line 248, in __stream
    auth=(self.user, self.password)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 490, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
2018-10-23 15:39:31,026 - ERROR: Bioagents - ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 440, in send
    timeout=timeout
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 639, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/util/retry.py", line 357, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 82, in respond_find_drugs_for_mutation_dataset
    result = self.BA.find_drugs_for_mutation_dataset(gene_list, dataset_arg.lower())
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_agent.py", line 221, in find_drugs_for_mutation_dataset
    for row in O.query().V(list(mut_samples[g])).in_("responseFor").mark("a").out("responseTo").mark("b").select(["a", "b"]):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/gripql/query.py", line 248, in __stream
    auth=(self.user, self.password)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 490, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
2018-10-23 15:39:31,026 - ERROR: Bioagents - ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 440, in send
    timeout=timeout
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 639, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/util/retry.py", line 357, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 82, in respond_find_drugs_for_mutation_dataset
    result = self.BA.find_drugs_for_mutation_dataset(gene_list, dataset_arg.lower())
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_agent.py", line 221, in find_drugs_for_mutation_dataset
    for row in O.query().V(list(mut_samples[g])).in_("responseFor").mark("a").out("responseTo").mark("b").select(["a", "b"]):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/gripql/query.py", line 248, in __stream
    auth=(self.user, self.password)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 490, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
2018-10-23 15:39:31,026 - ERROR: Bioagents - ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 440, in send
    timeout=timeout
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 639, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/util/retry.py", line 357, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 82, in respond_find_drugs_for_mutation_dataset
    result = self.BA.find_drugs_for_mutation_dataset(gene_list, dataset_arg.lower())
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_agent.py", line 221, in find_drugs_for_mutation_dataset
    for row in O.query().V(list(mut_samples[g])).in_("responseFor").mark("a").out("responseTo").mark("b").select(["a", "b"]):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/gripql/query.py", line 248, in __stream
    auth=(self.user, self.password)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 490, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
2018-10-23 15:39:31,026 - ERROR: Bioagents - ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 440, in send
    timeout=timeout
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 639, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/util/retry.py", line 357, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 82, in respond_find_drugs_for_mutation_dataset
    result = self.BA.find_drugs_for_mutation_dataset(gene_list, dataset_arg.lower())
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_agent.py", line 221, in find_drugs_for_mutation_dataset
    for row in O.query().V(list(mut_samples[g])).in_("responseFor").mark("a").out("responseTo").mark("b").select(["a", "b"]):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/gripql/query.py", line 248, in __stream
    auth=(self.user, self.password)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 490, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
2018-10-23 15:39:31,026 - ERROR: Bioagents - ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 440, in send
    timeout=timeout
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 639, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/util/retry.py", line 357, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 82, in respond_find_drugs_for_mutation_dataset
    result = self.BA.find_drugs_for_mutation_dataset(gene_list, dataset_arg.lower())
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_agent.py", line 221, in find_drugs_for_mutation_dataset
    for row in O.query().V(list(mut_samples[g])).in_("responseFor").mark("a").out("responseTo").mark("b").select(["a", "b"]):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/gripql/query.py", line 248, in __stream
    auth=(self.user, self.password)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 490, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
2018-10-23 15:39:31,053 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 15:39:31,053 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 15:39:31,053 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 15:39:31,053 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 15:39:31,053 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 15:39:31,053 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 15:39:31,053 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 15:39:31,053 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 15:39:31,053 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 15:39:31,053 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 15:39:31,053 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 15:39:31,053 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:00:11,553 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,559 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,559 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,564 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,564 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,564 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,570 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,570 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,570 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,570 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,575 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,575 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,575 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,575 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,575 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,580 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,580 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,580 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,580 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,580 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,580 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:00:11,595 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:00:11,595 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:00:11,595 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:00:11,595 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:00:11,595 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:00:11,595 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:00:11,595 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:00:11,595 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:00:11,595 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:00:11,595 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:00:11,595 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:00:11,595 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:05:57,564 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:05:57,564 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:05:57,564 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:05:57,564 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:05:57,564 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:05:57,564 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:05:57,569 - ERROR: Bioagents - ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 440, in send
    timeout=timeout
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 639, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/util/retry.py", line 357, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 82, in respond_find_drugs_for_mutation_dataset
    result = self.BA.find_drugs_for_mutation_dataset(gene_list, dataset_arg.lower())
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_agent.py", line 221, in find_drugs_for_mutation_dataset
    for row in O.query().V(list(mut_samples[g])).in_("responseFor").mark("a").out("responseTo").mark("b").select(["a", "b"]):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/gripql/query.py", line 248, in __stream
    auth=(self.user, self.password)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 490, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
2018-10-23 16:05:57,569 - ERROR: Bioagents - ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 440, in send
    timeout=timeout
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 639, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/util/retry.py", line 357, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 82, in respond_find_drugs_for_mutation_dataset
    result = self.BA.find_drugs_for_mutation_dataset(gene_list, dataset_arg.lower())
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_agent.py", line 221, in find_drugs_for_mutation_dataset
    for row in O.query().V(list(mut_samples[g])).in_("responseFor").mark("a").out("responseTo").mark("b").select(["a", "b"]):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/gripql/query.py", line 248, in __stream
    auth=(self.user, self.password)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 490, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
2018-10-23 16:05:57,569 - ERROR: Bioagents - ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 440, in send
    timeout=timeout
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 639, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/util/retry.py", line 357, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 82, in respond_find_drugs_for_mutation_dataset
    result = self.BA.find_drugs_for_mutation_dataset(gene_list, dataset_arg.lower())
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_agent.py", line 221, in find_drugs_for_mutation_dataset
    for row in O.query().V(list(mut_samples[g])).in_("responseFor").mark("a").out("responseTo").mark("b").select(["a", "b"]):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/gripql/query.py", line 248, in __stream
    auth=(self.user, self.password)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 490, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
2018-10-23 16:05:57,569 - ERROR: Bioagents - ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 440, in send
    timeout=timeout
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 639, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/util/retry.py", line 357, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 82, in respond_find_drugs_for_mutation_dataset
    result = self.BA.find_drugs_for_mutation_dataset(gene_list, dataset_arg.lower())
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_agent.py", line 221, in find_drugs_for_mutation_dataset
    for row in O.query().V(list(mut_samples[g])).in_("responseFor").mark("a").out("responseTo").mark("b").select(["a", "b"]):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/gripql/query.py", line 248, in __stream
    auth=(self.user, self.password)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 490, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
2018-10-23 16:05:57,569 - ERROR: Bioagents - ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 440, in send
    timeout=timeout
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 639, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/util/retry.py", line 357, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 82, in respond_find_drugs_for_mutation_dataset
    result = self.BA.find_drugs_for_mutation_dataset(gene_list, dataset_arg.lower())
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_agent.py", line 221, in find_drugs_for_mutation_dataset
    for row in O.query().V(list(mut_samples[g])).in_("responseFor").mark("a").out("responseTo").mark("b").select(["a", "b"]):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/gripql/query.py", line 248, in __stream
    auth=(self.user, self.password)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 490, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
2018-10-23 16:05:57,569 - ERROR: Bioagents - ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 440, in send
    timeout=timeout
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 639, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/util/retry.py", line 357, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/urllib3/connectionpool.py", line 383, in _make_request
    httplib_response = conn.getresponse()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 82, in respond_find_drugs_for_mutation_dataset
    result = self.BA.find_drugs_for_mutation_dataset(gene_list, dataset_arg.lower())
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_agent.py", line 221, in find_drugs_for_mutation_dataset
    for row in O.query().V(list(mut_samples[g])).in_("responseFor").mark("a").out("responseTo").mark("b").select(["a", "b"]):
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/gripql/query.py", line 248, in __stream
    auth=(self.user, self.password)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "/Users/durupina/.pyenv/versions/3.6.4/lib/python3.6/site-packages/requests/adapters.py", line 490, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
2018-10-23 16:05:57,717 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:05:57,717 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:05:57,717 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:05:57,717 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:05:57,717 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:05:57,717 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:05:57,719 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:05:57,719 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:05:57,719 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:05:57,719 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:05:57,719 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:05:57,719 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:06:32,931 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,936 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,936 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,940 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,940 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,940 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,947 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,947 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,947 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,947 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,952 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,952 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,952 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,952 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,952 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,956 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,956 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,956 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,956 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,956 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,956 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:06:32,970 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:06:32,970 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:06:32,970 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:06:32,970 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:06:32,970 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:06:32,970 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:06:32,970 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:06:32,970 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:06:32,970 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:06:32,970 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:06:32,970 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:06:32,970 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:07:14,505 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,506 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,506 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,508 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,508 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,508 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,511 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,511 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,511 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,511 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,513 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,513 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,513 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,513 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,513 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,515 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,515 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,515 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,515 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,515 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,515 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:07:14,521 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:07:14,521 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:07:14,521 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:07:14,521 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:07:14,521 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:07:14,521 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:07:14,521 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:07:14,521 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:07:14,521 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:07:14,521 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:07:14,521 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:07:14,521 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:09:06,221 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:09:06,221 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:09:06,221 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:09:06,221 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:09:06,221 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:09:06,221 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:09:06,221 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:09:06,221 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:09:06,221 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:09:06,221 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:09:06,221 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:09:06,221 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:13:11,701 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,706 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,706 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,710 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,710 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,710 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,716 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,716 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,716 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,716 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,721 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,721 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,721 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,721 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,721 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,726 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,726 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,726 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,726 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,726 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,726 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:13:11,738 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:13:11,738 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:13:11,738 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:13:11,738 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:13:11,738 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:13:11,738 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:13:11,739 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:13:11,739 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:13:11,739 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:13:11,739 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:13:11,739 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:13:11,739 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:14:43,164 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:14:43,164 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:14:43,164 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:14:43,164 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:14:43,164 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:14:43,164 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:14:43,165 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:14:43,165 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:14:43,165 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:14:43,165 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:14:43,165 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:14:43,165 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:19:39,121 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,126 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,126 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,130 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,130 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,130 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,137 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,137 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,137 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,137 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,142 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,142 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,142 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,142 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,142 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,146 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,146 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,146 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,146 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,146 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,146 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:19:39,160 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:19:39,160 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:19:39,160 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:19:39,160 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:19:39,160 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:19:39,160 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:19:39,160 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:19:39,160 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:19:39,160 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:19:39,160 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:19:39,160 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:19:39,160 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:21:33,146 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:21:33,146 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:21:33,146 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:21:33,146 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:21:33,146 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:21:33,146 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:21:33,146 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:21:33,146 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:21:33,146 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:21:33,146 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:21:33,146 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:21:33,146 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:22:30,407 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,409 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,409 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,411 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,411 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,411 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,415 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,415 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,415 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,415 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,417 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,417 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,417 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,417 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,417 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,419 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,419 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,419 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,419 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,419 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,419 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:30,424 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:22:30,424 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:22:30,424 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:22:30,424 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:22:30,424 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:22:30,424 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:22:30,424 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:22:30,424 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:22:30,424 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:22:30,424 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:22:30,424 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:22:30,424 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:22:34,024 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,029 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,029 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,033 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,033 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,033 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,040 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,040 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,040 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,040 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,045 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,045 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,045 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,045 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,045 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,049 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,049 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,049 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,049 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,049 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,049 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-23 16:22:34,061 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:22:34,061 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:22:34,061 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:22:34,061 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:22:34,061 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:22:34,061 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-23 16:22:34,061 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:22:34,061 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:22:34,061 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:22:34,061 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:22:34,061 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:22:34,061 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-23 16:23:34,095 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:23:34,095 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:23:34,095 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:23:34,095 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:23:34,095 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:23:34,095 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-23 16:23:34,096 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:23:34,096 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:23:34,096 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:23:34,096 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:23:34,096 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-23 16:23:34,096 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-24 12:58:59,687 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,693 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,693 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,697 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,697 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,697 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,703 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,703 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,703 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,703 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,708 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,708 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,708 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,708 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,708 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,715 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,715 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,715 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,715 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,715 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,715 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 12:58:59,730 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 12:58:59,730 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 12:58:59,730 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 12:58:59,730 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 12:58:59,730 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 12:58:59,730 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 12:58:59,731 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 12:58:59,731 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 12:58:59,731 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 12:58:59,731 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 12:58:59,731 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 12:58:59,731 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:00:05,181 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:00:05,181 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:00:05,181 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:00:05,181 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:00:05,181 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:00:05,181 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:00:05,183 - ERROR: Bioagents - 'list' object has no attribute 'to_string'
Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 94, in respond_find_drugs_for_mutation_dataset
    drugs = KQMLList.from_string(result.to_string())
AttributeError: 'list' object has no attribute 'to_string'
2018-10-24 13:00:05,183 - ERROR: Bioagents - 'list' object has no attribute 'to_string'
Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 94, in respond_find_drugs_for_mutation_dataset
    drugs = KQMLList.from_string(result.to_string())
AttributeError: 'list' object has no attribute 'to_string'
2018-10-24 13:00:05,183 - ERROR: Bioagents - 'list' object has no attribute 'to_string'
Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 94, in respond_find_drugs_for_mutation_dataset
    drugs = KQMLList.from_string(result.to_string())
AttributeError: 'list' object has no attribute 'to_string'
2018-10-24 13:00:05,183 - ERROR: Bioagents - 'list' object has no attribute 'to_string'
Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 94, in respond_find_drugs_for_mutation_dataset
    drugs = KQMLList.from_string(result.to_string())
AttributeError: 'list' object has no attribute 'to_string'
2018-10-24 13:00:05,183 - ERROR: Bioagents - 'list' object has no attribute 'to_string'
Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 94, in respond_find_drugs_for_mutation_dataset
    drugs = KQMLList.from_string(result.to_string())
AttributeError: 'list' object has no attribute 'to_string'
2018-10-24 13:00:05,183 - ERROR: Bioagents - 'list' object has no attribute 'to_string'
Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 94, in respond_find_drugs_for_mutation_dataset
    drugs = KQMLList.from_string(result.to_string())
AttributeError: 'list' object has no attribute 'to_string'
2018-10-24 13:00:05,197 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-24 13:00:05,197 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-24 13:00:05,197 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-24 13:00:05,197 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-24 13:00:05,197 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-24 13:00:05,197 - INFO: Bioagents - BMEGA received request with task: FIND-MUTATION-FREQUENCY
2018-10-24 13:00:05,198 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-24 13:00:05,198 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-24 13:00:05,198 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-24 13:00:05,198 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-24 13:00:05,198 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-24 13:00:05,198 - INFO: Bioagents - BMEGA will perform task FIND-MUTATION-FREQUENCY with method respond_find_mutation_frequency.
2018-10-24 13:01:35,131 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,136 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,136 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,141 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,141 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,141 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,148 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,148 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,148 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,148 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,153 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,153 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,153 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,153 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,153 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,158 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,158 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,158 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,158 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,158 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,158 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:01:35,173 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:01:35,173 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:01:35,173 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:01:35,173 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:01:35,173 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:01:35,173 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:01:35,173 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:01:35,173 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:01:35,173 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:01:35,173 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:01:35,173 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:01:35,173 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:04:39,871 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,876 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,876 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,880 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,880 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,880 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,887 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,887 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,887 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,887 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,892 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,892 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,892 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,892 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,892 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,898 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,898 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,898 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,898 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,898 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,898 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:04:39,910 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:04:39,910 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:04:39,910 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:04:39,910 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:04:39,910 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:04:39,910 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:04:39,910 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:04:39,910 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:04:39,910 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:04:39,910 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:04:39,910 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:04:39,910 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:05:24,951 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:05:24,951 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:05:24,951 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:05:24,951 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:05:24,951 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:05:24,951 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:05:24,956 - ERROR: Bioagents - {
Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 94, in respond_find_drugs_for_mutation_dataset
    drugs = KQMLList.from_string(drugs_str)
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_list.py", line 220, in from_string
    return kreader.read_list()
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_reader.py", line 163, in read_list
    raise KQMLBadOpenException(self.inbuf)
kqml.kqml_exceptions.KQMLBadOpenException: {
2018-10-24 13:05:24,956 - ERROR: Bioagents - {
Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 94, in respond_find_drugs_for_mutation_dataset
    drugs = KQMLList.from_string(drugs_str)
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_list.py", line 220, in from_string
    return kreader.read_list()
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_reader.py", line 163, in read_list
    raise KQMLBadOpenException(self.inbuf)
kqml.kqml_exceptions.KQMLBadOpenException: {
2018-10-24 13:05:24,956 - ERROR: Bioagents - {
Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 94, in respond_find_drugs_for_mutation_dataset
    drugs = KQMLList.from_string(drugs_str)
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_list.py", line 220, in from_string
    return kreader.read_list()
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_reader.py", line 163, in read_list
    raise KQMLBadOpenException(self.inbuf)
kqml.kqml_exceptions.KQMLBadOpenException: {
2018-10-24 13:05:24,956 - ERROR: Bioagents - {
Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 94, in respond_find_drugs_for_mutation_dataset
    drugs = KQMLList.from_string(drugs_str)
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_list.py", line 220, in from_string
    return kreader.read_list()
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_reader.py", line 163, in read_list
    raise KQMLBadOpenException(self.inbuf)
kqml.kqml_exceptions.KQMLBadOpenException: {
2018-10-24 13:05:24,956 - ERROR: Bioagents - {
Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 94, in respond_find_drugs_for_mutation_dataset
    drugs = KQMLList.from_string(drugs_str)
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_list.py", line 220, in from_string
    return kreader.read_list()
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_reader.py", line 163, in read_list
    raise KQMLBadOpenException(self.inbuf)
kqml.kqml_exceptions.KQMLBadOpenException: {
2018-10-24 13:05:24,956 - ERROR: Bioagents - {
Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 94, in respond_find_drugs_for_mutation_dataset
    drugs = KQMLList.from_string(drugs_str)
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_list.py", line 220, in from_string
    return kreader.read_list()
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_reader.py", line 163, in read_list
    raise KQMLBadOpenException(self.inbuf)
kqml.kqml_exceptions.KQMLBadOpenException: {
2018-10-24 13:05:25,055 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:05:25,055 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:05:25,055 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:05:25,055 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:05:25,055 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:05:25,055 - ERROR: Bioagents - Could not perform response to FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:05:25,056 - ERROR: Bioagents - {
Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 94, in respond_find_drugs_for_mutation_dataset
    drugs = KQMLList.from_string(drugs_str)
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_list.py", line 220, in from_string
    return kreader.read_list()
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_reader.py", line 163, in read_list
    raise KQMLBadOpenException(self.inbuf)
kqml.kqml_exceptions.KQMLBadOpenException: {
2018-10-24 13:05:25,056 - ERROR: Bioagents - {
Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 94, in respond_find_drugs_for_mutation_dataset
    drugs = KQMLList.from_string(drugs_str)
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_list.py", line 220, in from_string
    return kreader.read_list()
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_reader.py", line 163, in read_list
    raise KQMLBadOpenException(self.inbuf)
kqml.kqml_exceptions.KQMLBadOpenException: {
2018-10-24 13:05:25,056 - ERROR: Bioagents - {
Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 94, in respond_find_drugs_for_mutation_dataset
    drugs = KQMLList.from_string(drugs_str)
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_list.py", line 220, in from_string
    return kreader.read_list()
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_reader.py", line 163, in read_list
    raise KQMLBadOpenException(self.inbuf)
kqml.kqml_exceptions.KQMLBadOpenException: {
2018-10-24 13:05:25,056 - ERROR: Bioagents - {
Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 94, in respond_find_drugs_for_mutation_dataset
    drugs = KQMLList.from_string(drugs_str)
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_list.py", line 220, in from_string
    return kreader.read_list()
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_reader.py", line 163, in read_list
    raise KQMLBadOpenException(self.inbuf)
kqml.kqml_exceptions.KQMLBadOpenException: {
2018-10-24 13:05:25,056 - ERROR: Bioagents - {
Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 94, in respond_find_drugs_for_mutation_dataset
    drugs = KQMLList.from_string(drugs_str)
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_list.py", line 220, in from_string
    return kreader.read_list()
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_reader.py", line 163, in read_list
    raise KQMLBadOpenException(self.inbuf)
kqml.kqml_exceptions.KQMLBadOpenException: {
2018-10-24 13:05:25,056 - ERROR: Bioagents - {
Traceback (most recent call last):
  File "/Users/durupina/CWC/cwc-integ/hms/bioagents/bioagents/__init__.py", line 86, in _respond_to
    reply_content = resp(content)
  File "/Users/durupina/CWC/cwc-integ/hms/BMEGAgent/bmeg_agent/bmeg_module.py", line 94, in respond_find_drugs_for_mutation_dataset
    drugs = KQMLList.from_string(drugs_str)
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_list.py", line 220, in from_string
    return kreader.read_list()
  File "/Users/durupina/CWC/cwc-integ/hms/pykqml/kqml/kqml_reader.py", line 163, in read_list
    raise KQMLBadOpenException(self.inbuf)
kqml.kqml_exceptions.KQMLBadOpenException: {
2018-10-24 13:07:07,859 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:07:07,866 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:07:07,866 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:07:07,871 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:07:07,871 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:07:07,871 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:07:07,871 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:09:15,646 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:09:15,653 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:09:15,653 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:09:15,658 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:09:15,658 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:09:15,658 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:09:15,658 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:12:59,045 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:12:59,052 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:12:59,052 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:12:59,057 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:12:59,057 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:12:59,058 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:12:59,058 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:15:20,693 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:15:20,696 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:15:20,696 - INFO: Bioagents - BMEGA has started and is ready.
2018-10-24 13:15:20,698 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:15:20,698 - INFO: Bioagents - BMEGA received request with task: FIND-DRUGS-FOR-MUTATION-DATASET
2018-10-24 13:15:20,698 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
2018-10-24 13:15:20,698 - INFO: Bioagents - BMEGA will perform task FIND-DRUGS-FOR-MUTATION-DATASET with method respond_find_drugs_for_mutation_dataset.
